docs/api/doc-block-title.md

```diff
@@ -24,6 +24,12 @@ import { Title } from '@storybook/blocks';
 
 `Title` is configured with the following props:
 
+### `of`
```

Comment:
Thanks for updating the API reference! ❤️ 

Could you please keep these props in alphabetical order?